db/wide/wide_columns_helper.h

```diff
@@ -0,0 +1,23 @@
+//  Copyright (c) Meta Platforms, Inc. and affiliates.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+#include <ostream>
+#include <string>
+
+#include "rocksdb/rocksdb_namespace.h"
+#include "rocksdb/wide_columns.h"
+
+namespace ROCKSDB_NAMESPACE {
+class WideColumnsHelper {
+ public:
+  WideColumnsHelper() = default;
+
```

Comment:
Very minor but there's no need for declaring a constructor - this class is just a collection of static helper methods (so there's no need for users to actually construct any instances of it).